fix: Customization Custom Action (#653)
ZupIT/beagle-docs: content/en/ios/customization/custom-actions/_index.md

---
title: Custom Action
weight: 141
description: >-

>   In this section, you will find the description of how to create a custom action and

Reviewer: ```suggestion
  In this section, you will find the description on how to create a custom action and
```

  details of the methods it implements.
---

---

## Introduction

> In Beagle, an action deals with the behaviors (functions) that will be executed in your application as soon as a certain event is triggered. These actions can be Beagle's own standard or custom actions.

Reviewer: ```suggestion
An action in Beagle is a function that will be executed in your application when a certain event is triggered. Beagle Actions can be either **default** or **custom**
```
